plugins/techdocs/src/reader/components/OverviewContent.tsx

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
I wonder if it makes sense to put this and other homepage components out of "reader/components/" directory. 🤔

Reply:
yeah I was thinking about that actually. I'll take the opportunity to move out all components from reader in this PR as well!

Reply:
Nice!